.github/actions/filter-files/action.yml

```diff
@@ -0,0 +1,37 @@
+name: 'Filter files'
+description: 'Filter the list of changed files'
+inputs:
+  changed-files:
+    description: 'Space-separated list of changed files from setup'
```

Comment:
So, it definitely doesn't. I'm mostly queueing off of wonder-stuff's actions. The other option is to pass around a json-serialized list, which is somewhat more annoying to pass around, but probably better practice 😬

Reply:
I'll do some exploring in a separate PR about how to most elegantly protect against filenames with spaces 😬  good ol' bash argument syntax is going to be a bear

Reply:
For sure. I don't want to send you off on a rabbit hole. I'm not sure that we even have spaces in paths in any of our repos... so this might be a non-issue. 🤷‍♂️